### PR TITLE
Update CustomDataLoaderOnSL.ipynb

### PR DESCRIPTION
**Pull request by zoeStartover**
issue #699

### CODE REVIEW

**Comment by zoeStartover** [comment]
I have read the CLA Document and I hereby sign the CLA

File: docs/tutorial/CustomDataLoaderOnSL.ipynb [hunk 1]
```diff
@@ -76,108 +95,81 @@
...
-    "在SplitLearning中（各方数据需要对齐）我们暂时仅提供了面向CSV数据的DataBuilder能力。在自定义DataBuilder中可以定义对每一行数据怎么处理。SLModel会根据您定义的方式进行操作。\n",
+    "We customize a DataBuilder with the aim to meet higher level customization demands, as the existing FedDataFrame and FedNdarray did not provide the required functionality.\n",
     "\n",
-    "需要注意的点:  \n",
-    "- 我们的MovieLens数据集读取进来后会是CSV格式的，但是需要被当做稀疏特征来进行处理，所以会在DataBuilder中将每一列转成字典形式\n",
-    "- 在Bob测我们将评分进行了Threshold处理成二值形式\n",
-    "- 在DataBuilder中可以定义自定义函数，然后使用dataset.map来应用到每一列。\n",
-    "- 因为垂直模式需要保证两方数据是对齐的，所以暂时只支持CSV模式\n",
-    "- CSV模式只需要返回Dataset，Dataset需要定义好Batch_size，Repeat，SLModel会根据Dataset推断Steps_per_epoch"
+    "In SplitLearning(where all sides of the data need to be aligned), we only provide DataBuilder for CSV data for the time being. You can define what to do with each row of data in the custom DataBuilder. The operations in SLModel depend on the way you define it.\n",
```
Comment (Mingbo-Lee) [inline docs/tutorial/CustomDataLoaderOnSL.ipynb:136]: SplitLearning -> Split Learning.
later same in Chinese translation.
It is a error from the previous operation.

File: docs/tutorial/CustomDataLoaderOnSL.ipynb [hunk 2]
```diff
@@ -76,108 +95,81 @@
...
+    "\n",
+    "The things you should notice here:\n",
+    "- We read MovieLens dataset in CSV format. However, it needs to be treated as sparse features. We should convert each column in the DataBuilder to dictionary form.\n",
+    "- Bob's score was processed into binary form using threshold.\n",
+    "- Custom functions can be defined in the DataBuilder and then applied to each column using dataset.map.\n",
```
Comment (Mingbo-Lee) [inline docs/tutorial/CustomDataLoaderOnSL.ipynb:141]: `We should convert each column in the DataBuilder to dictionary form` --> `The DataBuilder will convert each column into the form of dictionary`

File: docs/tutorial/CustomDataLoaderOnSL.ipynb [hunk 3]
```diff
@@ -76,108 +95,81 @@
...
+    "- Only CSV mode is supported in the vertical mode due to the restriction that both sides of the data should be aligned. \n",
```
Comment (Mingbo-Lee) [inline docs/tutorial/CustomDataLoaderOnSL.ipynb:142]: `Only CSV mode is supported in the vertical mode due to the restriction that both sides of the data should be aligned`
-> `Only CSV format is supported in the vertical scenario due to the restriction that both sides of the data should be aligned`

File: docs/tutorial/CustomDataLoaderOnSL.ipynb [hunk 4]
```diff
@@ -76,108 +95,81 @@
...
    ]
   },
   {
-   "attachments": {},
    "cell_type": "markdown",
-   "id": "e79c519b",
    "metadata": {},
    "source": [
-    "### 1、下载数据，并转换成csv"
+    "### 1. Download the dataset and convert it to a CSV format."
```
Comment (Mingbo-Lee) [inline docs/tutorial/CustomDataLoaderOnSL.ipynb:151]: `convert it to a CSV format` --> `convert it to the CSV format`

File: docs/tutorial/CustomDataLoaderOnSL.ipynb [hunk 5]
```diff
@@ -76,108 +95,81 @@
...
-  },
   {
    "cell_type": "code",
-   "execution_count": 4,
-   "id": "6d050edf",
+   "execution_count": 9,
    "metadata": {},
    "outputs": [],
    "source": [
+    "# Read the data from the dat format and convert it to a dictionary form\n",
```
Comment (Mingbo-Lee) [inline docs/tutorial/CustomDataLoaderOnSL.ipynb:172]: No `#`
`convert it to a dictionary form` -> `convert it to the dictionary form`

File: docs/tutorial/CustomDataLoaderOnSL.ipynb [hunk 6]
```diff
@@ -231,18 +223,15 @@
    ]
   },
   {
-   "attachments": {},
    "cell_type": "markdown",
-   "id": "c4ee4435",
    "metadata": {},
    "source": [
-    "接下来我们将user、movie和rating进行join，并进行拆分，组装成`alice_ml1m.csv`和`bob_ml1m.csv`"
+    "Next, we join user, movie and rating. Split them and assemble them into alice_ml1m.csv and bob_ml1m.csv."
```
Comment (Mingbo-Lee) [inline docs/tutorial/CustomDataLoaderOnSL.ipynb:229]: `. Split` -> `, split`.
(`) which includes the name of the file should be retained.

File: docs/tutorial/CustomDataLoaderOnSL.ipynb [hunk 7]
```diff
@@ -309,23 +296,15 @@
...
-   "id": "d1a0534b",
    "metadata": {},
    "source": [
-    "## 到此为止我们已经完成了数据的处理和拆分\n",
-    "产出了\n",
-    "```\n",
-    "alice: alice_ml1m.csv\n",
-    "bob: bob_ml1m.csv\n",
-    "```"
+    "So we've done the data processing and splitting up by now. And we output two files: *alice_ml1m.csv, bob_ml1m.csv*."
```
Comment (Mingbo-Lee) [inline docs/tutorial/CustomDataLoaderOnSL.ipynb:302]: `So we've done the data processing and splitting up by now. And we output two files: *alice_ml1m.csv, bob_ml1m.csv*.`
`## So we've done the data processing and splitting up by now. And we output two files:  *Alice: alice_ml1m.csv* and *Bob: bob_ml1m.csv*.`

File: docs/tutorial/CustomDataLoaderOnSL.ipynb [hunk 8]
```diff
@@ -377,28 +355,29 @@
    ]
   },
   {
-   "attachments": {},
    "cell_type": "markdown",
-   "id": "0cb771fe",
    "metadata": {},
    "source": [
-    "### 1. 使用明文引擎开发DataBuilder函数  \n",
```
Comment (Mingbo-Lee) [inline docs/tutorial/CustomDataLoaderOnSL.ipynb:385]: please keep the numeric sequence number correspondence the same.

File: docs/tutorial/CustomDataLoaderOnSL.ipynb [hunk 9]
```diff
@@ -377,28 +355,29 @@
...
+    "### 2. Use plaintext engines to develop Databuilders."
    ]
   },
   {
-   "attachments": {},
    "cell_type": "markdown",
-   "id": "3ff04282",
    "metadata": {},
    "source": [
-    "#### 开发Alice侧的DataBuilder"
```
Comment (Mingbo-Lee) [inline docs/tutorial/CustomDataLoaderOnSL.ipynb:395]: `Develop Alice's DataBuilder`->`#### Develop Alice's DataBuilder`

File: docs/tutorial/CustomDataLoaderOnSL.ipynb [hunk 10]
```diff
@@ -598,18 +587,15 @@
    ]
   },
   {
-   "attachments": {},
    "cell_type": "markdown",
-   "id": "8c3878e1",
    "metadata": {},
    "source": [
-    "#### 开发Bob侧的DataBuilder\n"
```
Comment (Mingbo-Lee) [inline docs/tutorial/CustomDataLoaderOnSL.ipynb:606]: `Develop Bob's DataBuilder` ->`#### Develop Bob's DataBuilder`

File: docs/tutorial/CustomDataLoaderOnSL.ipynb [hunk 11]
```diff
@@ -923,18 +905,15 @@
    ]
   },
   {
-   "attachments": {},
    "cell_type": "markdown",
-   "id": "2ef012c0",
    "metadata": {},
    "source": [
-    "### 2. 将单方的DataBuilder 包装(wrap)起来"
```
Comment (Mingbo-Lee) [inline docs/tutorial/CustomDataLoaderOnSL.ipynb:931]: 1. please keep the numeric sequence number correspondence the same.
2. `wrap`->`Wrap`

File: docs/tutorial/CustomDataLoaderOnSL.ipynb [hunk 12]
```diff
@@ -994,18 +973,15 @@
    ]
   },
   {
-   "attachments": {},
    "cell_type": "markdown",
-   "id": "45465838",
    "metadata": {},
    "source": [
-    "### 3.构造databuilder_dict"
```
Comment (Mingbo-Lee) [inline docs/tutorial/CustomDataLoaderOnSL.ipynb:1002]: `4. Construct a databuilder_dict.` ->`3. Construct databuilder_dict.`

File: docs/tutorial/CustomDataLoaderOnSL.ipynb [hunk 13]
```diff
@@ -1022,27 +998,22 @@
    ]
   },
   {
-   "attachments": {},
    "cell_type": "markdown",
-   "id": "89d14244",
    "metadata": {},
    "source": [
-    "## 接下来我们定义DeepFM模型，将数据跑起来"
```
Comment (Mingbo-Lee) [inline docs/tutorial/CustomDataLoaderOnSL.ipynb:1030]: `DeepFM Model` -> `Define DeepFM Model and run it`

File: docs/tutorial/CustomDataLoaderOnSL.ipynb [hunk 14]
```diff
@@ -1076,18 +1047,15 @@
    ]
   },
   {
-   "attachments": {},
    "cell_type": "markdown",
-   "id": "14119ef5",
    "metadata": {},
    "source": [
-    "### 定义alice的basenet"
```
Comment (Mingbo-Lee) [inline docs/tutorial/CustomDataLoaderOnSL.ipynb:1084]: `BaseNet` -> `basenet`

File: docs/tutorial/CustomDataLoaderOnSL.ipynb [hunk 15]
```diff
@@ -1146,18 +1114,15 @@
    ]
   },
   {
-   "attachments": {},
    "cell_type": "markdown",
-   "id": "0bf27e66",
    "metadata": {},
    "source": [
```
Comment (Mingbo-Lee) [inline docs/tutorial/CustomDataLoaderOnSL.ipynb:1119]: `BaseNet` -> `basenet`, `FuseNet` -> `fusenet`

**Review by Mingbo-Lee — COMMENTED** [review]
LGTM.
Please wait an approval from a viewer who has access to write, and then continue to commit the Chinese translation.

File: docs/tutorial/CustomDataLoaderOnSL.ipynb [hunk 16]
```diff
@@ -21,28 +24,40 @@
...
+    "pycharm": {
+     "name": "#%% md\n"
+    }
+   },
    "source": [
-    "隐语在SLModel中提供了联邦学习在垂直场景的支持，本文将以DeepFM推荐场景为例介绍下，如何在SLModel中使用自定义DataBuilder  \n",
-    "这篇文档的主要目标是借助DeepFM训练来介绍如何在SLModel中使用自定义DataBuilder  \n",
-    "*注：DeepFM相关的内容以及拆分方案，请移步DeepFM相关文档*"
+    "We support federated learning in vertical scenarios on SLModel in SecretFlow. In this tutorial, we will take DeepFM model for recommendation as an example to introduce how to customize a DataBuilder on SLModel.\n",
+    "\n",
```
Comment (zhangxingmeng) [inline docs/tutorial/CustomDataLoaderOnSL.ipynb:34]: You can build english tutorial here and see if the rendering effect is correct。
```sh docs/build.sh -l en```

Comment (Mingbo-Lee) [reply docs/tutorial/CustomDataLoaderOnSL.ipynb:34]: hi! @zhangxingmeng  this command would got error.
```
(sf) limingbo@DESKTOP-mingbo:/mnt/d/WorkShop/temp/secretflow$ sh docs/build.sh -l en
: not foundsh: 2:
docs/build.sh: 7: Syntax error: word unexpected (expecting "in")
(sf) limingbo@DESKTOP-mingbo:/mnt/d/WorkShop/temp/secretflow$ cd docs/
```
maybe we could refer to https://www.yuque.com/secret-flow/admin/uwv3kw683w24czs8 to generate the html
```
make SPHINXOPTS="-D language='en'" html
```

Comment (zhangxingmeng) [reply docs/tutorial/CustomDataLoaderOnSL.ipynb:34]: of course, We just want to make sure that the rendering effect is correct, especially line breaks.

Comment (Mingbo-Lee) [reply docs/tutorial/CustomDataLoaderOnSL.ipynb:34]: When you finish this action, resolve this conversation.

**Comment by zoeStartover** [comment]
hi,[@Ming](https://github.com/secretflow/secretflow/pull/705#discussion_r1251683747),
origin version(Chinese):
图片
new version(English):
图片

**Comment by Mingbo-Lee** [comment]
LGTM

File: docs/tutorial/CustomDataLoaderOnSL.ipynb [hunk 17]
```diff
@@ -78,83 +110,70 @@
...
-   "id": "798f117c",
-   "metadata": {},
+   "metadata": {
+    "pycharm": {
+     "name": "#%% md\n"
+    }
+   },
    "source": [
-    "我们对数据进行了切分：\n",
+    "We will use the classic dataset [MovieLens](https://secretflow-data.oss-accelerate.aliyuncs.com/datasets/movielens/ml-1m.zip) for introduction.MovieLens is an open recommendation system dataset that contains movie ratings and movie metadata information.\n",
```
Comment (zhangxingmeng) [inline docs/tutorial/CustomDataLoaderOnSL.ipynb:127]: please format：add a space after punctuation marks

**Comment by Mingbo-Lee** [comment]
Hi, @zoeStartover, Good for your English translation.  .please commit your po file and the screenshot of the Chinese version.

File: docs/locales/zh_CN/LC_MESSAGES/tutorial/CustomDataLoaderOnSL.po [hunk 18]
```diff
@@ -3,154 +3,175 @@
 # This file is distributed under the same license as the SecretFlow package.
 # FIRST AUTHOR <EMAIL@ADDRESS>, 2023.
 #
+#, fuzzy
```
Comment (Mingbo-Lee) [inline docs/locales/zh_CN/LC_MESSAGES/tutorial/CustomDataLoaderOnSL.po:6]: ref to https://github.com/secretflow/secretflow/blob/main/CONTRIBUTING.md#documentation-update-and-its-multilingual-version 
all `#, fuzzy` need to be removed.

File: docs/locales/zh_CN/LC_MESSAGES/tutorial/CustomDataLoaderOnSL.po [hunk 19]
```diff
@@ -3,154 +3,175 @@
...
 "MIME-Version: 1.0\n"
 "Content-Type: text/plain; charset=utf-8\n"
 "Content-Transfer-Encoding: 8bit\n"
 "Generated-By: Babel 2.12.1\n"
 
 #: ../../tutorial/CustomDataLoaderOnSL.ipynb:9
-msgid "在SecretFlow中使用DataBuilder进行SLModel学习"
-msgstr ""
+msgid "Customize DataBuilder on SLModel in SecretFlow"
+msgstr "在SecretFlow中使用DataBuilder进行SLModel学习"
```
Comment (Mingbo-Lee) [inline docs/locales/zh_CN/LC_MESSAGES/tutorial/CustomDataLoaderOnSL.po:24]: Is there need to add a blankspace between Chinese word and English word in the Chinese document? such as 
`在SecretFlow中使用DataBuilder进行SLModel学习`->`在 SecretFlow 中使用 DataBuilder 进行 SLModel 学习`
@zhangxingmeng

File: docs/locales/zh_CN/LC_MESSAGES/tutorial/CustomDataLoaderOnSL.po [hunk 20]
```diff
@@ -3,154 +3,175 @@
...
 
 #: ../../tutorial/CustomDataLoaderOnSL.ipynb:20
 msgid ""
-"The following codes are demos only. It’s **NOT for production** due to "
+"The following codes are demos only. It's **NOT for production** due to "
 "system security concerns, please **DO NOT** use it directly in "
 "production."
-msgstr ""
+msgstr "下面的代码只是演示。由于系统安全问题,请不要直接在生产中使用它。"
```
Comment (Mingbo-Lee) [inline docs/locales/zh_CN/LC_MESSAGES/tutorial/CustomDataLoaderOnSL.po:32]: `在生产中`-->`**在生产中**`
Please check it's effect of bold, if not then don't change.

File: docs/locales/zh_CN/LC_MESSAGES/tutorial/CustomDataLoaderOnSL.po [hunk 21]
```diff
@@ -3,154 +3,175 @@
...
-
-#: ../../tutorial/CustomDataLoaderOnSL.ipynb:32
-msgid "这篇文档的主要目标是借助DeepFM训练来介绍如何在SLModel中使用自定义DataBuilder"
-msgstr ""
+msgid ""
+"We support federated learning in vertical scenarios on SLModel in "
+"SecretFlow. In this tutorial, we will take DeepFM model for "
+"recommendation as an example to introduce how to customize a DataBuilder "
+"on SLModel."
+msgstr "隐语在SLModel中提供了联邦学习在垂直场景的支持，本文将以DeepFM推荐场景为例介绍下，如何在SLModel中使用自定义DataBuilder。"
```
Comment (Mingbo-Lee) [inline docs/locales/zh_CN/LC_MESSAGES/tutorial/CustomDataLoaderOnSL.po:39]: 本文将以DeepFM推荐场景为例介绍~下~

File: docs/locales/zh_CN/LC_MESSAGES/tutorial/CustomDataLoaderOnSL.po [hunk 22]
```diff
@@ -3,154 +3,175 @@
...
-msgid "*注：DeepFM相关的内容以及拆分方案，请移步DeepFM相关文档*"
-msgstr ""
-
-#: ../../tutorial/CustomDataLoaderOnSL.ipynb:45
-msgid "环境设置"
-msgstr ""
+msgid ""
+"The main goal of this tutorial is to train DeepFM to show how to "
+"customize a DataBuilder on SLModel."
+msgstr "这篇文档的主要目标是借助DeepFM训练来介绍如何在SLModel中使用自定义DataBuilder。"
```
Comment (Mingbo-Lee) [inline docs/locales/zh_CN/LC_MESSAGES/tutorial/CustomDataLoaderOnSL.po:45]: 文档--> 教程

File: docs/locales/zh_CN/LC_MESSAGES/tutorial/CustomDataLoaderOnSL.po [hunk 23]
```diff
@@ -3,154 +3,175 @@
...
 
-#: ../../tutorial/CustomDataLoaderOnSL.ipynb:108
-msgid "数据集介绍"
-msgstr ""
+#: ../../tutorial/CustomDataLoaderOnSL.ipynb:35
+msgid ""
+"*If you want to learn more about related content for DeepFM model and "
+"split plans, please move to related documents for DeepFM model.*"
+msgstr "注：DeepFM相关的内容以及拆分方案，请移步DeepFM相关文档"
```
Comment (Mingbo-Lee) [inline docs/locales/zh_CN/LC_MESSAGES/tutorial/CustomDataLoaderOnSL.po:52]: DeepFM相关的内容以及拆分方案--> 如果您想了解DeepFM相关的内容以及拆分方案

File: docs/locales/zh_CN/LC_MESSAGES/tutorial/CustomDataLoaderOnSL.po [hunk 24]
```diff
@@ -3,154 +3,175 @@
...
+#: ../../tutorial/CustomDataLoaderOnSL.ipynb:175
 msgid ""
-"`数据集地址 <https://secretflow-data.oss-"
-"accelerate.aliyuncs.com/datasets/movielens/ml-1m.zip>`__"
-msgstr ""
+"We will use the classic dataset `MovieLens <https://secretflow-data.oss-"
+"accelerate.aliyuncs.com/datasets/movielens/ml-1m.zip>`__ for "
+"introduction. MovieLens is an open recommendation system dataset that "
+"contains movie ratings and movie metadata information."
+msgstr "我们这里将使用最经典的MovieLens数据集来进行演示。MovieLens是一个开放式的推荐系统数据集，包含了电影评分和电影元数据信息。"
```
Comment (Mingbo-Lee) [inline docs/locales/zh_CN/LC_MESSAGES/tutorial/CustomDataLoaderOnSL.po:67]: 最经典 -> 经典；
Please add the reference link of the dataset in Chinese translation.

File: docs/locales/zh_CN/LC_MESSAGES/tutorial/CustomDataLoaderOnSL.po [hunk 25]
```diff
@@ -3,154 +3,175 @@
...
+" score was processed into binary form using threshold. - Custom functions"
+" can be defined in the DataBuilder and then applied to each column using "
+"dataset.map. - Only CSV format is supported in the vertical scenario due "
+"to the restriction that both sides of the data should be aligned. - It "
+"returns Dataset which has been defined Batch\\_size and Repeat in the CSV"
+" mode so that SLModel can infer the Steps\\_per\\_epoch according to the "
+"Dataset."
+msgstr "需要注意的点:""
+"我们的MovieLens数据集读取进来后会是CSV格式的，但是需要被当做稀疏特征来进行处理，所以会在DataBuilder中将每一列转成字典形式"
+"在Bob测我们将评分进行了Threshold处理成二值形式"
```
Comment (Mingbo-Lee) [inline docs/locales/zh_CN/LC_MESSAGES/tutorial/CustomDataLoaderOnSL.po:109]: Bob测 --> Bob侧；
Threshold->阈值（Threshold）

File: docs/locales/zh_CN/LC_MESSAGES/tutorial/CustomDataLoaderOnSL.po [hunk 26]
```diff
@@ -3,154 +3,175 @@
...
+#: ../../tutorial/CustomDataLoaderOnSL.ipynb:1529
+msgid "3. Construct a databuilder\\_dict."
+msgstr "3.构造databuilder_dict"
 
-#: ../../tutorial/CustomDataLoaderOnSL.ipynb:1501
-msgid "接下来我们定义DeepFM模型，将数据跑起来"
-msgstr ""
+#: ../../tutorial/CustomDataLoaderOnSL.ipynb:1560
+msgid "Define DeepFM Model and run it"
+msgstr "接下来我们定义DeepFM模型，将数据跑起来"
```
Comment (Mingbo-Lee) [inline docs/locales/zh_CN/LC_MESSAGES/tutorial/CustomDataLoaderOnSL.po:164]: `接下来我们定义DeepFM模型，将数据跑起来`->`接下来我们定义DeepFM模型，并且运行模型`

File: docs/locales/zh_CN/LC_MESSAGES/tutorial/CustomDataLoaderOnSL.po [hunk 27]
```diff
@@ -3,154 +3,175 @@
...
+#: ../../tutorial/CustomDataLoaderOnSL.ipynb:883
+msgid "Develop Bob's DataBuilder"
+msgstr "开发Bob侧的DataBuilder"
 
-#: ../../tutorial/CustomDataLoaderOnSL.ipynb:1396
-msgid "2. 将单方的DataBuilder 包装(wrap)起来"
-msgstr ""
+#: ../../tutorial/CustomDataLoaderOnSL.ipynb:1455
+msgid "2. Wrap their DataBuilders separately."
+msgstr "2. 将单方的DataBuilder 包装(wrap)起来"
```
Comment (Mingbo-Lee) [inline docs/locales/zh_CN/LC_MESSAGES/tutorial/CustomDataLoaderOnSL.po:157]: `(wrap)`-->`（wrap）`

File: docs/locales/zh_CN/LC_MESSAGES/tutorial/CustomDataLoaderOnSL.po [hunk 28]
```diff
@@ -3,154 +3,175 @@
...
+#: ../../tutorial/CustomDataLoaderOnSL.ipynb:1572
+msgid "1. Define a DeepFM model."
+msgstr "定义DeepFM模型"
 
-#: ../../tutorial/CustomDataLoaderOnSL.ipynb:1561
-msgid "定义alice的basenet"
-msgstr ""
+#: ../../tutorial/CustomDataLoaderOnSL.ipynb:1620
+msgid "2. Define Alice's basenet."
+msgstr "定义alice的basenet"
```
Comment (Mingbo-Lee) [inline docs/locales/zh_CN/LC_MESSAGES/tutorial/CustomDataLoaderOnSL.po:172]: alice -> Alice

File: docs/locales/zh_CN/LC_MESSAGES/tutorial/CustomDataLoaderOnSL.po [hunk 29]
```diff
@@ -3,154 +3,175 @@
...
+
+#: ../../tutorial/CustomDataLoaderOnSL.ipynb:1693
+msgid "3.Define Bob's basenet and fusenet."
+msgstr "定义bob的basenet和fusenet"
```
Comment (Mingbo-Lee) [inline docs/locales/zh_CN/LC_MESSAGES/tutorial/CustomDataLoaderOnSL.po:176]: bob-->Bob

**Comment by Mingbo-Lee** [comment]
LGTM

**Comment by zhangxingmeng** [comment]
You also need to add a PO file to provide the Chinese version.

File: docs/tutorial/CustomDataLoaderOnSL.ipynb [hunk 30]
```diff
@@ -233,16 +250,14 @@
   {
    "attachments": {},
    "cell_type": "markdown",
-   "id": "c4ee4435",
    "metadata": {},
    "source": [
-    "接下来我们将user、movie和rating进行join，并进行拆分，组装成`alice_ml1m.csv`和`bob_ml1m.csv`"
+    "Next, we join user, movie and rating, split up and assemble them into alice_ml1m.csv and bob_ml1m.csv. "
```
Comment (zhangxingmeng) [inline docs/tutorial/CustomDataLoaderOnSL.ipynb:255]: 英文版本的“`”符号还是带上吧

**Review by zhangxingmeng — COMMENTED** [review]
整体看下来没什么大问题，麻烦确认下渲染的网页效果是正常的就可以合并了。
尤其是列表的换行部分。

**Comment by zoeStartover** [comment]
> You also need to add a PO file to provide the Chinese version.

Thank you for your review. I have already added the corresponding PO file and you can check in file changed. I am not sure you refers to that PO file.

**Review by zhangxingmeng — APPROVED** [review]
LGTM

**Pull request merged**